geode-core/src/main/java/org/apache/geode/internal/cache/RegionEntry.java

```diff
@@ -461,4 +461,8 @@ boolean destroy(InternalRegion region, EntryEventImpl event, boolean inTokenMode
   @Retained(ABSTRACT_REGION_ENTRY_PREPARE_VALUE_FOR_CACHE)
   Object prepareValueForCache(RegionEntryContext context, Object value, EntryEventImpl event,
       boolean isEntryUpdate);
+
+  default boolean isEvictBitSet() {
```

Comment:
Could you named this "isEvicted" instead of "isEvictBitSet"? Then you could use the existing isEvicted in the LRU code and just add @Override to it. Also it is okay for "isEvicted" to be on more than one interface.
Did you run into a problem with naming it isEvicted?

Reply:
It will get compile-time error as class can not inherits abstract and default for isEvicted().

Reply:
Seems we need to override every abstract LRU region entry class the method and return false work around this compile-time error.

Reply:
Maybe the problem is that it's trying to inherit the default version and the abstract version from different directions. Could we create an `Evictable` interface with `setEvicted()`, `unsetEvicted()`, and `default isEvicted()`, then make `RegionEntry` and `EvictionNode` extend that? That way the abstract LRU classes that implement `isEvictec()` would override the default one, rather than offering a conflicting version that subclasses can't resolve. (Note: I don't know that this will work.)